src/components/form/FormFieldOptionGroup.vue

```diff
@@ -0,0 +1,183 @@
+<script lang="ts">
+/**
+ * FormFieldOptionGroup Component
+ *
+ * The `FormFieldOptionGroup` displays option group input.
+ *
+ * @description * Use this component to render option groups with icon title
+ * and description in the form of cards.
+ *
+ * Note: This component is commonly used in `FormRegisterChallenge`.
+ *
+ * @props
+ * - `modelValue` (string, required): The object representing user input.
+ * - `name` (string, required): The name used for id and test selectors.
+ * - `label` (string, required): The translation key for the label.
+ *
+ * @events
+ * - `update:modelValue`: Emitted as a part of v-model structure.
+ *
+ * @example
+ * <form-field-option-group v-model="value" />
+ *
+ * @see [Figma Design](https://www.figma.com/file/L8dVREySVXxh3X12TcFDdR/Do-pr%C3%A1ce-na-kole?type=design&node-id=5350%3A24169&mode=dev)
+ */
+
+// libraries
+import { colors } from 'quasar';
+import { defineComponent, computed } from 'vue';
+
+// composables
+import { useValidation } from 'src/composables/useValidation';
+
+// types
+import { ConfigGlobal } from '../types';
+
+// config
+const rideToWorkByBikeConfig: ConfigGlobal = JSON.parse(
```

Comment:
Here boot/global_vars.ts we have declared and parsed global config. Do you really need to parse the JSON configuration here again?

https://github.com/auto-mat/ride-to-work-by-bike-frontend/blob/a52a67d85e78a1f847ccf58e7165cb297797ab3f/src/boot/global_vars.ts#L8-L10

Reply:
No, you are correct. We can import directly from `import { rideToWorkByBikeConfig } from 'src/boot/global_vars';`